<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Meet Your Faculty | Metabolomics 2023</title>
  <meta name="description" content="Workshop website for the 2023 Metabolomics Canadian Bioinformatics Workshop" />
  <meta name="generator" content="bookdown 0.43 and GitBook 2.6.7" />

  <meta property="og:title" content="Meet Your Faculty | Metabolomics 2023" />
  <meta property="og:type" content="book" />
  <meta property="og:image" content="https://bioinformaticsdotca.github.io/MET_2023/img/bioinformatics_logo.png" />
  <meta property="og:description" content="Workshop website for the 2023 Metabolomics Canadian Bioinformatics Workshop" />
  <meta name="github-repo" content="bioinformaticsdotca/MET_2023" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Meet Your Faculty | Metabolomics 2023" />
  
  <meta name="twitter:description" content="Workshop website for the 2023 Metabolomics Canadian Bioinformatics Workshop" />
  <meta name="twitter:image" content="https://bioinformaticsdotca.github.io/MET_2023/img/bioinformatics_logo.png" />

<meta name="author" content="Faculty: David Wishart, Jianguo Xia, Zhiqiang Pang, Jessica Ewald, Lun Zhang, Mark Berkjanskii, Michelle Brazas, and Nia Hughes" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon" />
<link rel="prev" href="index.html"/>
<link rel="next" href="data.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css">



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://bioinformaticsdotca.github.io/">
 <img src="img/bioinformatics_logo.png" width="90%" alt="bioinformatics.ca logo" style="display:block; margin-left:auto; margin-right:auto; margin-top:15px;">
 </a>
<li><center><strong><a href="./"><br>Metabolomics<br><br></a></strong></center></li>
<a href="./">
 <img src="img/metabolomics_icon.png" width="70%" alt="Metabolomics icon" style="display:block; margin-left:auto; margin-right:auto; ">
 </a>
<br>

<li class="divider"></li>
<li class="part"><span><b>I Introduction</b></span></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Workshop Info</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#schedule"><i class="fa fa-check"></i>Schedule</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#edmonton"><i class="fa fa-check"></i>Edmonton</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#montréal"><i class="fa fa-check"></i>Montréal</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#pre-work"><i class="fa fa-check"></i>Pre-work</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="meet-your-faculty.html"><a href="meet-your-faculty.html"><i class="fa fa-check"></i>Meet Your Faculty</a></li>
<li class="chapter" data-level="" data-path="data.html"><a href="data.html"><i class="fa fa-check"></i>Data</a></li>
<li class="part"><span><b>II Modules</b></span></li>
<li class="chapter" data-level="" data-path="module-1-introduction-to-metabolomics.html"><a href="module-1-introduction-to-metabolomics.html"><i class="fa fa-check"></i>Module 1: Introduction to Metabolomics</a>
<ul>
<li class="chapter" data-level="" data-path="module-1-introduction-to-metabolomics.html"><a href="module-1-introduction-to-metabolomics.html#lecture"><i class="fa fa-check"></i>Lecture</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-2-targeted-quantitative-metabolomics.html"><a href="module-2-targeted-quantitative-metabolomics.html"><i class="fa fa-check"></i>Module 2: Targeted, Quantitative Metabolomics</a>
<ul>
<li class="chapter" data-level="" data-path="module-2-targeted-quantitative-metabolomics.html"><a href="module-2-targeted-quantitative-metabolomics.html#lecture-1"><i class="fa fa-check"></i>Lecture</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-3-quantitative-metabolomics.html"><a href="module-3-quantitative-metabolomics.html"><i class="fa fa-check"></i>Module 3: Quantitative Metabolomics</a>
<ul>
<li class="chapter" data-level="" data-path="module-3-quantitative-metabolomics.html"><a href="module-3-quantitative-metabolomics.html#lab"><i class="fa fa-check"></i>Lab</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-4-databases-for-biological-interpretation.html"><a href="module-4-databases-for-biological-interpretation.html"><i class="fa fa-check"></i>Module 4: Databases for Biological Interpretation</a>
<ul>
<li class="chapter" data-level="" data-path="module-4-databases-for-biological-interpretation.html"><a href="module-4-databases-for-biological-interpretation.html#lecture-2"><i class="fa fa-check"></i>Lecture</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-5-backgroun-in-omics-data-science.html"><a href="module-5-backgroun-in-omics-data-science.html"><i class="fa fa-check"></i>Module 5: Backgroun in Omics Data Science</a>
<ul>
<li class="chapter" data-level="" data-path="module-5-backgroun-in-omics-data-science.html"><a href="module-5-backgroun-in-omics-data-science.html#lecture-3"><i class="fa fa-check"></i>Lecture</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-6-data-analytics-for-untargeted-metabolomics.html"><a href="module-6-data-analytics-for-untargeted-metabolomics.html"><i class="fa fa-check"></i>Module 6: Data Analytics for Untargeted Metabolomics</a>
<ul>
<li class="chapter" data-level="" data-path="module-6-data-analytics-for-untargeted-metabolomics.html"><a href="module-6-data-analytics-for-untargeted-metabolomics.html#lecture-4"><i class="fa fa-check"></i>Lecture</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-7-metabolomics-data-analysis-using-metaboanalyst-5.html"><a href="module-7-metabolomics-data-analysis-using-metaboanalyst-5.html"><i class="fa fa-check"></i>Module 7: Metabolomics Data Analysis using MetaboAnalyst 5.0</a>
<ul>
<li class="chapter" data-level="" data-path="module-7-metabolomics-data-analysis-using-metaboanalyst-5.html"><a href="module-7-metabolomics-data-analysis-using-metaboanalyst-5.html#lab-1"><i class="fa fa-check"></i>Lab</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="module-8-integrating-metabolomics-with-other-omics.html"><a href="module-8-integrating-metabolomics-with-other-omics.html"><i class="fa fa-check"></i>Module 8: Integrating Metabolomics with Other Omics</a>
<ul>
<li class="chapter" data-level="" data-path="module-8-integrating-metabolomics-with-other-omics.html"><a href="module-8-integrating-metabolomics-with-other-omics.html#lecture-5"><i class="fa fa-check"></i>Lecture</a></li>
</ul></li>
<li class="divider"></li>
<h1 id="sponsors">Sponsors</h1>
<center>
<a href="https://genomecanada.ca/"><img src="img/sponsors/genomecanada.png" height="70" style="display:inline-block;"  style="padding-bottom: 20px" alt="Genome Canada logo.">
<a href="https://www.ontariogenomics.ca/"><img src="img/sponsors/ontariogenomics.png" height="70" style="display:inline-block;" style="padding-bottom: 20px;" alt="Ontario Genomics logo.">
<br><br>
<a href="https://oicr.on.ca/"><img src="img/sponsors/oicr.png" width=25% style="display:inline-block;" style="padding-bottom: 20px" alt="Ontario Institute for Cancer Research logo.">
<a href="https://www.hpcforhealth.ca/"><img src="img/sponsors/hpc4health.png" width=60% style="display:inline-block;" style="padding-bottom: 20px "alt="HPC4Health logo.">
<br><br>
<a href="https://computationalgenomics.ca/"><img src="img/sponsors/computationalgenomics.png" style="display:inline-block;" width=35% style="padding-bottom: 20px" alt="Canadian Centre for Computational Genomcis logo.">
<a href="https://www.mdpi.com/journal/metabolites?ci=28767&si=23914&_utm_from=f6c8c977b1"><img src="img/sponsors/metabolites.png" style="display:inline-block;" width=35% style="padding-bottom: 20px" alt="Metabolites logo.">
<br><br>
<a href="https://metabolomicscentre.ca/"><img src="img/sponsors/metabolicscentre.png" style="display:inline-block;" width=35% style="padding-bottom: 20px" alt="Metabolics Centre logo.">

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Metabolomics 2023</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="meet-your-faculty" class="section level1 hasAnchor">
<h1>Meet Your Faculty<a href="meet-your-faculty.html#meet-your-faculty" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="david-wishart" class="section level4 hasAnchor">
<h4>David Wishart<a href="meet-your-faculty.html#david-wishart" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/david-wishart.png" width="200" alt="Photo of David Wishart"></p>
<blockquote>
<p>Distinguished University Professor of Biological Science and Computing Science <br>
University of Alberta <br>
Edmonton, AB, CA <br></p>
<p>— <a href="mailto:dwishart@ualberta.ca" class="email">dwishart@ualberta.ca</a> <br>
<a href="www.wishartlab.com">www.wishartlab.com</a></p>
</blockquote>
<p>David was one of the co-founders of the CBW in 1998. He has active research interests in the
application of bioinformatics and statistics to a wide range of computational biology problems,
from genomics to proteomics to metabolomics. In addition to running a large computational
biology lab, David also operates a large wet lab (analytical chemistry, molecular biology,
nanotechnology, cell biology, structural biology) supported by its own fabrication shop and
electrical engineering group.</p>
</div>
<div id="jianguo-jeff-xia" class="section level4 hasAnchor">
<h4>Jianguo (Jeff) Xia<a href="meet-your-faculty.html#jianguo-jeff-xia" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/jianguo-xia.png" width="200" alt="Photo of Jianguo Xia"></p>
<blockquote>
<p>Canada Research Chair in Bioinformatics and Big Data Analysis <br>
Institute of Parasitology, McGill University <br>
Montréal, QC, CA <br></p>
<p>— <a href="mailto:jeff.xia@cmgill.ca" class="email">jeff.xia@cmgill.ca</a> <br>
<a href="www.xialab.ca">www.xialab.ca</a></p>
</blockquote>
<p>Jeff has a broad background in medicine, immunology and bioinformatics. He has been involved
in CBW since 2011. Jeff has active research interests in metabolomics, transcriptomics,
microbiomics and multi-omics integration, with a key focus on democratizing omics data
analytics via web/cloud - based platforms and visualization.</p>
</div>
<div id="zhiqiang-pang" class="section level4 hasAnchor">
<h4>Zhiqiang Pang<a href="meet-your-faculty.html#zhiqiang-pang" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/zhiqiang-pang.png" width="200" alt="Photo of Zhiqiang Pang"></p>
<blockquote>
<p>PhD Candidate <br>
Institute of Parasitology, McGill University <br>
Montréal, QC, CA <br></p>
<p>— <a href="mailto:zhiqiang.pang@mail.mcgill.ca" class="email">zhiqiang.pang@mail.mcgill.ca</a></p>
</blockquote>
<p>Zhiqiang is a PhD candidate in the Xia Lab at McGill University.
He graduated from Jilin University with his Bachelor’s degree from
School of Pharmaceutical Sciences (2016) and Master’s degree from the College of Basic
Medical Sciences (2019). His research project is about optimizing metabolomics raw data
processing to gain deep biological insight on infectious diseases, cancer and host-microbiome
interactions.</p>
</div>
<div id="jessica-ewald" class="section level4 hasAnchor">
<h4>Jessica Ewald<a href="meet-your-faculty.html#jessica-ewald" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/jessica-ewald.png" width="200" alt="Photo of Jessica Ewald"></p>
<blockquote>
<p>Postdoctoral Fellow <br>
Institute of Parasitology, McGill University <br>
Montréal, QC, CA <br></p>
<p>— <a href="mailto:jessica.ewald@mail.mcgill.ca" class="email">jessica.ewald@mail.mcgill.ca</a></p>
</blockquote>
<p>Jessica is a postdoctoral fellow in the Xia Lab at McGill University
where she is currently studying multi-omics integration. Her
background includes an undergraduate degree in Environmental
Engineering and Applied Mathematics (Harvard College, 2017) and
a PhD in Bioinformatics (McGill University, 2022). Overall, her research has focused on using
computational approaches to understand the impact of environmental contaminants on human
and ecosystem health.</p>
</div>
<div id="lun-allen-zhang" class="section level4 hasAnchor">
<h4>Lun (Allen) Zhang<a href="meet-your-faculty.html#lun-allen-zhang" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/lun-zhang.png" width="200" alt="Photo of Lun Zhang"></p>
<blockquote>
<p>Research Associate <br>
Department of the Biological Sciences, University of Alberta <br>
Edmonton, AB, CA <br></p>
<p>— <a href="mailto:lun2@ualberta.ca" class="email">lun2@ualberta.ca</a></p>
</blockquote>
<p>Dr. Lun Zhang is a Research Associate in the Wishart lab at the
University of Alberta. Since joining the lab in 2017, he has mainly
focused on the development of LC-MS-based metabolomics
assays and software. He also participated in multiple translational
medicine research projects, such as discovering the diagnostic biomarkers for lung cancer,
colorectal cancer and polyps.</p>
</div>
<div id="mark-berjanskii" class="section level4 hasAnchor">
<h4>Mark Berjanskii<a href="meet-your-faculty.html#mark-berjanskii" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/mark-berjanskii.png" width="200" alt="Photo of Mark Berjanskii"></p>
<blockquote>
<p>Research Associate <br>
Department of the Biological Sciences, University of Alberta <br>
Edmonton, AB, CA <br></p>
<p>— <a href="mailto:mb1@ualberta.ca" class="email">mb1@ualberta.ca</a></p>
</blockquote>
<p>Mark obtained a Ph.D. in Biochemistry at University of
Missouri-Columbia, USA. Mark’s research interests include NMR
metabolomics, bioinformatics, protein NMR structure determination,
interactions, misfolding, and dynamics. Between 1996 and 2004, he
worked as a member of several research teams that studied the Hsp70-Hsp40 chaperone
system. Mark joined Dr. Wishart’s group at University of Alberta in 2004. Between 2005 and
2013, Mark studied prion proteins that, when misfolded, cause Mad Cow Disease in cattle and
similar Creutzfeldt-Jakob Disease in humans. Since 2004, he has been involved in developing
several programs for analysis of protein structure and dynamics, such as Random Coil Index,
Preditor, GeNMR, CS23D, PROSESS, Resolution-by-proxy, and Gamdy, as well as
bioinformatic analysis and NMR metabolomics.</p>
</div>
<div id="michelle-brazas" class="section level4 hasAnchor">
<h4>Michelle Brazas<a href="meet-your-faculty.html#michelle-brazas" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/michelle-brazas.jpg" width="200" alt="Photo of Michelle Brazas"></p>
<blockquote>
<p>Acting Scientific Director <br>
Canadian Bioinformatics Workshop (CBW) <br>
Toronto, ON, CA <br></p>
<p>— <a href="mailto:supporting@bioinformatics.ca" class="email">supporting@bioinformatics.ca</a></p>
</blockquote>
<p>Dr. Michelle Brazas is the Associate Director for Adaptive Oncology at
the Ontario Institute for Cancer Research (OICR), and acting Scientific
Director at Bioinformatics.ca. Previously, Dr. Brazas was the Program Manager for
Bioinformatics.ca and a faculty member in Biotechnology at BCIT. Michelle co-founded and runs
the Toronto Bioinformatics User Group (TorBUG) now in its 11th season, and plays an active
role in the International Society of Computational Biology where she sits on the Board of
Directors and Executive Board.</p>
</div>
<div id="nia-hughes" class="section level4 hasAnchor">
<h4>Nia Hughes<a href="meet-your-faculty.html#nia-hughes" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><img src="img/faculty/nia-hughes.jpeg" width="200" alt="Photo of Nia Hughes"></p>
<blockquote>
<p>Program Manager, Bioinformatics.ca <br>
Ontario Institute for Cancer Research (CBW) <br>
Toronto, ON, CA <br></p>
<p>— <a href="mailto:nia.hughes@oicr.on.ca" class="email">nia.hughes@oicr.on.ca</a></p>
</blockquote>
<p>Nia is the Program Manager for Bioinformatics.ca, where she coordinates
the Canadian Bioinformatics Workshop Series. Prior to starting at OICR, she completed her
M.Sc. in Bioinformatics from the University of Guelph in 2020 before working there as a
bioinformatician studying epigenetic and transcriptomic patterns across maize varieties.</p>

<!--
IF YOUR WORKSHOP DOES NOT INCLUDE COMPUTE:
- REMOVE "AND COMPUTE SETUP" FROM THE TOP-LEVEL HEADER
- DELETE THE LAST SECTION (COMPUTE SETUP)

IF YOUR WORKSHOP INCLUDES COMPUTE:
- AFTER THE WORKSHOP, ADD DATA DOWNLOAD LINKS, AWS MACHINE IMAGE INSTRUCTIONS, ETC.
- IF IT IS MORE APPROPRIATE, YOU CAN INCLUDE THE DATA DOWNLOAD LINKS IN EACH OF THE MODULES.
-->
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="data.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": "https://github.com/bioinformaticsdotca/MET_2023/blob/main/001-faculty.Rmd",
    "text": "Edit"
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": ["_main.pdf", "_main.epub"],
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

</body>

</html>
